meson_options.txt

```diff
@@ -317,6 +317,8 @@ option('systemd-resolve-uid', type : 'integer', value : 0,
        description : 'soft-static allocation for the systemd-resolve user')
 option('systemd-timesync-uid', type : 'integer', value : 0,
        description : 'soft-static allocation for the systemd-timesync user')
+option('suppress-sysusers', type : 'string',
+       description : 'list of users/groups to omit in sysusers config')
```

Comment:
Actually, things will work fine if the list is whitespace separate too. I had this info initially in the description, and I removed it, because it's not really needed (we have comma-separate lists pretty much everywhere, so people will just use that), and meson doesn't deal with long descriptions very well. So I think it's better to keep this implicit.

Also, I don't want to mention `basic.conf`, because it's possible that we could use the same option for groups in other files later on (e.g. for some groups needed by udev).